sources/PTYTab.m

```diff
@@ -1891,15 +1886,66 @@ - (double)blurRadius
     }
 }
 
-- (void)recheckBlur
-{
+- (BOOL)useInactiveTransparency {
+    NSArray* sessions = [self sessions];
+    for (PTYSession* session in sessions) {
+        if ([[session textview] useInactiveTransparency]) {
+            return YES;
+        }
+    }
+    return NO;
+}
+
+- (BOOL)inactiveBlur {
+    NSArray* sessions = [self sessions];
+    for (PTYSession* session in sessions) {
+        if ([session inactiveTransparency] > 0 &&
+            [[session textview] useInactiveTransparency] &&
+            [[[session profile] objectForKey:KEY_INACTIVE_BLUR] boolValue]) {
+            return YES;
+        }
+    }
+    return NO;
+}
+
+- (double)inactiveBlurRadius {
```

Comment:
There's a lot of duplicated code with -blurRadius. Please refactor into a common method like
`- (double)averageBlurRadiusForInactive:(BOOL)inactive`